### ISSUE TITLE
increased sputum production

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6720]
**Preferred term label:**
increased sputum production

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
COVID-23  33/76  PMID: 33289374

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
+1

**Comment by pnrobinson** [comment]
@mgriese is this definition good?


An increase in the amount of airway mucus, an extracellular gel in which water and mucins (heavily glycosylated proteins) are the most important components. This feature may be characterized by frequent or excessive throat clearing (exhalation through tightly constricted laryngopharyngeal tissues accompanied by vibration of the palatoglossal arch and the vocal folds serving to clear mucus from the airway). 
PMID: 16983448

Comment: If excessive sputum production is manifested by frequent and often involuntary coughing, the HPO term Productuve cough should be used.

synonym: excessive production of airway mucus, hypersecretion of airway mucus

**Comment by pnrobinson** [comment]
An increase in the amount of airway mucus. This feature may be characterized by frequent or excessive throat clearing (exhalation through tightly constricted laryngopharyngeal tissues accompanied by vibration of the palatoglossal arch and the vocal folds serving to clear mucus from the airway).

**Comment by pnrobinson** [comment]
added
